### PR TITLE
chore(cli): update deps, fix log compilation issue

### LINKED ISSUE #9026: [bug] cargo install tauri-cli fails due to log 0.4.21 update
### Describe the bug

there was a new `log` patch release a few hours ago which removed the Value.to_str api. Compare https://docs.rs/log/0.4.21/log/kv/struct.Value.html with https://docs.rs/log/0.4.20/log/kv/struct.Value.html which results in an error like this one
```
error[E0599]: no method named `to_str` found for struct `log::kv::Value` in the current scope
   --> /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tauri-cli-1.5.10/src/lib.rs:132:29
    |
132 |         let action = action.to_str().unwrap();
    |                             ^^^^^^ help: there is a method with a similar name: `to_cow_str`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `tauri-cli` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `tauri-cli v1.5.10`, intermediate artifacts can be found at `/tmp/cargo-installFdAUrt`.
```
(from https://discord.com/channels/616186924390023171/1212539466548510740)

I guess that's what we get for using an unstable feature flag 😅 

Is it possible to remove the unstable api usage somehow? Alternatively, we can just lock `log` to 0.4.20 in the cli of course 🤷 

### Reproduction

_No response_

### Expected behavior

_No response_

### Full `tauri info` output

```text
i'm on my phone right now :)
```

### Stack trace

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by lucasfernog**
log pushed a breaking change on a patch release: https://github.com/rust-lang/log/pull/592

### CODE REVIEW

File: examples/api/vite.config.js.timestamp-1709208499973-5c44d403dbeea.mjs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// vite.config.js
```
Comment (FabianLars) [inline examples/api/vite.config.js.timestamp-1709208499973-5c44d403dbeea.mjs]: i assume this file was an accident?

Comment (lucasfernog) [reply examples/api/vite.config.js.timestamp-1709208499973-5c44d403dbeea.mjs]: aaaa why does this happen all the time

**Review by FabianLars — APPROVED** [review]

**Comment by Thomasdezeeuw** [comment]
I would suggest to switch to the stable `kv` feature. The currently used `kv_unstable` will be removed in a couple of months.

**Comment by amrbashir** [comment]
thanks for the heads up @Thomasdezeeuw , will migrate asap

**Comment by drernie** [comment]
is this fix available in a release yet? How can we tell?

**Comment by amrbashir** [comment]
Not yet, but soon for sure. For now, try `cargo install tauri-cli --version "^2.0.0-beta" --locked`

**Comment by CaoKha** [comment]
> Not yet, but soon for sure. For now, try `cargo install tauri-cli --version "^2.0.0-beta" --locked`

What about `tauri-cli` stable version (1.5)? Do you have any solution ?

**Comment by FabianLars** [comment]
@CaoKha `cargo install tauri-cli --locked`

**Comment by CaoKha** [comment]
@FabianLars  I got this
```bash
error[E0635]: unknown feature `stdsimd`
  --> /home/username/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ahash-0.8.6/src/lib.rs:99:42
   |
99 | #![cfg_attr(feature = "stdsimd", feature(stdsimd))]
   |                                          ^^^^^^^
For more information about this error, try `rustc --explain E0635`.
error: could not compile `ahash` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `tauri-cli v1.5.10`, intermediate artifacts can be found at `/tmp/cargo-installIo79UU`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

**Comment by CaoKha** [comment]
actually, it compiles fine with rust stable version. The error I got is from rust nightly version set up in `rust-toolchain` file.

**Comment by Thomasdezeeuw** [comment]
> is this fix available in a release yet? How can we tell?

The `kv` feature was released in v0.4.21, the same one that contained this breakage of unstable API. It's documented in the changelog.

**Pull request merged**